.changeset/unlucky-points-matter.md

```diff
@@ -2,4 +2,4 @@
 "@comet/cms-api": minor
 ---
 
-API Generator: Add support for filtering many-to-many-relations by id
+API Generator: Add support for filtering many-to-many and one-to-many relations by id
```

Comment:
I'd prefer if this would be an additional changeset because then the link in the release would link to the correct commit (and therefore PR). Or is this neglectable?